emsdk

```diff
@@ -88,6 +88,14 @@ ENABLE_WASM = False
 ENABLE_LLVM_ASSERTIONS = 'auto'
 
 
+def os_name():
+  if WINDOWS: return 'win'
+  elif LINUX: return 'linux'
+  elif OSX: return 'osx'
```

Comment:
Our bot is called `mac` not `osx`. So unfortunately we may need separate functions for moz vs emscripten-releases/waterfall URLs.

Reply:
Turns out this is ok, since the urls for emscripten-releases and other things are hardcoded in the manifest file (and I did use `mac` there).

But if we need to refer to those urls from emsdk, we'll need to generalize `os_name`, yeah.